docs/pages/divider.js

```diff
@@ -1,64 +1,388 @@
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+          ]}
+        />
+      )}
+    </Flex>
+  );
+
+  return (
+    <Flex direction="column" gap={10} width={300}>
+      <Block title="Home decor" url="https://i.ibb.co/121JJzC/stock7.jpg" locked/>
+      <Divider />
+      <Block title="Plants" url="https://i.ibb.co/FY2MKr5/stock6.jpg" />
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Inset the Divider in a way that causes it to be free-floating or separated from content."
+            defaultCode={`
+function Example() {
+  const Block = ({ title, url, locked }) => (
+    <Flex gap={2} alignItems="center">
+      <Box maxWidth={60}>
+        <Mask rounding={4}>
+          <img alt="example image" src={url} style={{ maxWidth: '100%', display: 'block' }} />
+        </Mask>
+      </Box>
+      <Flex.Item flex="grow">
+        <Text weight="bold" size="md">
+          {title}
+        </Text>
+      </Flex.Item>
+      {locked ? (
+        <Icon accessibilityLabel="" icon="lock" size={12} />
+      ) : (
+        <AvatarGroup
+          accessibilityLabel="Collaborators."
+          size="sm"
+          collaborators={[
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+          ]}
+        />
+      )}
+    </Flex>
+  );
+
+  return (
+    <Flex direction="column" gap={10} width={300}>
+      <Block title="Home decor" url="https://i.ibb.co/121JJzC/stock7.jpg" locked/>
+      <Flex width="100%" justifyContent="center">
+        <Box width="50%">
+          <Divider />
+        </Box>
+      </Flex>
+      <Block title="Plants" url="https://i.ibb.co/FY2MKr5/stock6.jpg" />
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Dividers to separate collections of content or create the appearance of containers."
+            defaultCode={`
+function Example() {
+  return (
+    <Flex width="100%" height="100%">
+      <Box paddingX={4}>
+        <Flex direction="column" gap={2} width={120}>
+        {["Public profile", "Account settings", "Home feed tuner", "Claim", "Permissions", "Notifications"].map((item, idx) => <Text underline={idx === 0}weight="bold" key={idx} size="md" wrap="none">{item}</Text>)}
```

Comment:
The underline here is a bit odd…I know it's supposed to indicate the currently-selected item, but it reads as a link, so it's unexpected that it's not a link. Perhaps just remove the underline? Or implement using a Box similar to how it's implemented in Pinboard.

Reply:
DONE, changed to Tabs